File: Examples/OSX/AudioKitDemo/AudioKitDemo/Synthesis/TouchView.m
//
//  TouchView.m
//  AudioKitDemo
//
//  Created by Aurelius Prochazka on 2/23/15.
//  Copyright (c) 2015 Aurelius Prochazka. All rights reserved.
//

#import "TouchView.h"

@implementation TouchView

- (void)drawRect:(NSRect)dirtyRect {
    // set any NSColor for filling, say white:
    [[NSColor colorWithCalibratedRed:0.090 green:0.671 blue:0.094 alpha:1.000] setFill];
    NSRectFill(dirtyRect);
    [super drawRect:dirtyRect];
}

@end

File: Examples/OSX/Swift/AudioKitDemo/AudioKitDemo/TouchView.swift
//
//  TouchView.swift
//  AudioKitDemo
//
//  Created by Nicholas Arner on 3/13/15.
//  Copyright (c) 2015 AudioKit. All rights reserved.
//


class TouchView: NSView {
    
    override func drawRect(dirtyRect: NSRect) {
        NSColor(calibratedRed: 0.090, green: 0.671, blue: 0.094, alpha: 1.000).setFill()
        NSRectFill(dirtyRect)
        super.drawRect(dirtyRect)
    }
}